src/Modules/ASayModule/Controllers/ASayController.cs

```diff
@@ -0,0 +1,38 @@
+﻿using EvoSC.Commands;
+using EvoSC.Commands.Attributes;
+using EvoSC.Common.Controllers;
+using EvoSC.Common.Controllers.Attributes;
+using EvoSC.Modules.Official.ASayModule.Interfaces;
+
+namespace EvoSC.Modules.Official.ASayModule.Controllers;
+
+[Controller]
+public class ASayController : EvoScController<CommandInteractionContext>
+{
+    private readonly IASayService _asayService;
+
+    // You want to dependency inject the needed services here at the constructor
```

Comment:
Don't think this comment is necessary here.

Reply:
fixed